ui/src/pages/cancerScreening.jsx

```diff
@@ -0,0 +1,220 @@
+import {
+  Container,
+  TextField,
+  Stack,
+  Button,
+  Grid,
+  Snackbar,
+  Typography,
+  Divider,
+  useMediaQuery,
+  Radio,
+  RadioGroup,
+  Alert,
+  FormControlLabel,
+  FormLabel,
+} from "@mui/material";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import Layout from "../components/Layout";
+import { getCookie } from "../lib/cookie";
+import Tab from "@mui/material/Tab";
+import TabContext from "@mui/lab/TabContext";
+import TabList from "@mui/lab/TabList";
+import TabPanel from "@mui/lab/TabPanel";
+import { Box, FormControl, InputLabel, Select, MenuItem } from "@mui/material";
+import { DesktopDatePicker } from "@mui/x-date-pickers/DesktopDatePicker";
+import { MobileDatePicker } from "@mui/x-date-pickers/MobileDatePicker";
+import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import CurrentPatient from "../components/CurrentPatient";
+import { useFormik } from "formik";
+import * as yup from "yup";
+import Preview from "../components/Preview";
+import FormFields from "../components/FormFields";
+import cancerScreeningFields from "../lib/forms/cancerScreening";
+import { apiHost, createEncounter, FhirApi } from "./../lib/api";
+import cancerScreeningValidationSchema from "../lib/forms/validations/cancerScreeningValidation";
+
+export default function CancerScreening() {
+  let navigate = useNavigate();
+  let [open, setOpen] = useState(false);
+  let [visit, setVisit] = useState();
+  let [message, setMessage] = useState(false);
+  let isMobile = useMediaQuery("(max-width:600px)");
+  const [value, setValue] = useState("1");
+  const [inputData, setInputData] = useState({});
+  const [preview, setPreview] = useState(false);
+
+  const fieldValues = Object.values(cancerScreeningFields).flat();
+  console.log("cancerScreeningFields:", cancerScreeningFields);
+
+  const initialValues = Object.assign(
+      {},
+      ...fieldValues.map((item) => ({ [item.name]: "" }))
+  );
+
+  const formik = useFormik({
+      initialValues: {
+          ...initialValues,
+      },
+      validationSchema: cancerScreeningValidationSchema,
+      onSubmit: (values) => {
+          //alert("clicked");
```

Comment:
My bad, here is an example of the commented lines of code

Reply:
Will do